llvm/lib/Support/CommandLine.cpp

```diff
@@ -175,6 +175,24 @@ class CommandLineParser {
                                StringRef Overview, raw_ostream *Errs = nullptr,
                                bool LongOptionsUseDoubleDash = false);
 
+  void forEachSubCommand(Option &Opt,
+                         std::function<void(SubCommand &)> Action) {
```

Comment:
Please use function_ref instead of std::function here.

Reply:
Thanks for the suggestion. I've created #75973 for this.